src/site/content/en/blog/samesite-cookie-recipes/index.md

```diff
@@ -0,0 +1,324 @@
+---
+title: SameSite cookie recipes
+subhead:
+  Update your site's cookies to prepare for the upcoming changes to the SameSite
+  attribute's behavior.
+authors:
+  - rowan_m
+date: 2019-10-15
+hero: cookie-hero.jpg
+description: |
+  With the introduction of the new SameSite=None attribute value, sites can now
+  explicitly mark their cookies for cross-site usage. Browsers are moving to
+  make cookies without a SameSite attribute act as first-party by default, a
+  safer and more privacy preserving option than the current open behavior.
+  Learn how to mark up your cookies to ensure your first-party and third-party
+  cookies continue to work once this change comes into effect.
+tags:
+  - post
+  - security
+  - cookies
+---
+
+{% Aside %}
+
+If you need a refresher on cookies and the `SameSite` attribute, head over to
+the earlier companion article,
+["SameSite cookies explained"](/samesite-cookies-explained).
+
+{% endAside %}
+
+To recap from before: [Chrome](https://www.chromium.org/updates/same-site),
```

Comment:
Delete 'from before'. 

The reason for this is the same as the previous suggestion, [Strunk and White](https://www.amazon.com/Elements-Style-Fourth-William-Strunk/dp/020530902X/ref=pd_sbs_14_t_0/140-9010431-1453962?_encoding=UTF8&pd_rd_i=020530902X&pd_rd_r=f53e256d-a283-4b76-997c-5c58c53af430&pd_rd_w=1qObN&pd_rd_wg=sszWF&pf_rd_p=5cfcfe89-300f-47d2-b1ad-a4e27203a02a&pf_rd_r=JWVHE4V0RS2BJEFACQVD&psc=1&refRID=JWVHE4V0RS2BJEFACQVD) rule II 17, "Omit needless words."

Reply:
By the way, there's another rule, V 19, "Do not take shortcuts at the cost of clarity." It doesn't seem to be a problem here. I mention it because much of technical writing is about finding the happy path between the two rules.

Reply:
Recap removed.